test/development/acceptance-app/__snapshots__/ReactRefreshLogBox.test.ts.snap

```diff
@@ -149,12 +149,9 @@ exports[`ReactRefreshLogBox app turbo conversion to class component (1) 1`] = `
 `;
 
 exports[`ReactRefreshLogBox app turbo css syntax errors 1`] = `
-"./index.module.css:1:1
+"./index.module.css
 Parsing css source code failed
-> 1 | button {}
-    | ^^^^^^
-
-Selector is not pure (pure selectors must contain at least one local class or id)"
+Selector is not pure (pure selectors must contain at least one local class or id), (lightningcss, Selector(button, specificity = 0x1))"
```

Comment:
This is missing line and column info

Reply:
`lightingcss` does not provide span information because it is backed by `cssparser` and `selectors`, which are developed by the servo team.